references/classification/fastai/train.py

```diff
@@ -71,21 +75,20 @@ def main(args):
                         help='number of data loading workers (default: 16)')
     parser.add_argument('--lr', default=3e-3, type=float, help='initial learning rate')
     parser.add_argument("--concat-pool", dest="concat_pool",
-        help="Use pre-trained models from the modelzoo",
-        action="store_true"
-    )
+                        help="Use pre-trained models from the modelzoo",
+                        action="store_true")
     parser.add_argument('--dropout-prob', default=0.5, type=float, help='dropout rate of last FC layer')
     parser.add_argument('--wd', '--weight-decay', default=1e-2, type=float,
                         metavar='W', help='weight decay (default: 1e-2)',
                         dest='weight_decay')
     parser.add_argument('--div-factor', default=25., type=float, help='div factor of OneCycle policy')
     parser.add_argument('--checkpoint', default='checkpoint', type=str, help='name of output file')
-    parser.add_argument(
-        "--pretrained",
-        dest="pretrained",
-        help="Use pre-trained models from the modelzoo",
-        action="store_true",
-    )
+    parser.add_argument('--resume', default=None, help='resume from checkpoint')
```

Comment:
It's a detail but the way the help is written, the use may think it is a yes/no while the user has to provide the name of the checkpoint if I understand well.